docs/website/docs/building-from-source/getting-started.md

```diff
@@ -138,32 +144,102 @@ cmake --build ../iree-build/
 
     See also our [developer documentation for ccache](https://github.com/openxla/iree/blob/main/docs/developers/developing_iree/ccache.md).
 
-## What's next?
+### Optional components
+
+By default, the CMake build includes:
+
+* All compiler targets (`llvm-cpu`, `cuda`, `vulkan-spirv`, etc.)
+* All runtime HAL drivers (`local-task`, `cuda`, `vulkan`, etc.)
+* All compiler input formats (StableHLO, TOSA, etc.)
+* All compiler output formats (VM bytecode, C)
 
-<!-- TODO(scotttodd): "at this point you can..." -->
+The default build does _not_ include:
+
+* Compiler or runtime bindings (Python, TFLite, etc.)
+* Advanced features like AddressSanitizer or tracing instrumentation
+* Experimental components
+
+These can be changed via the `IREE_` CMake options listed in the root
+[`CMakeLists.txt`](https://github.com/openxla/iree/blob/main/CMakeLists.txt).
+
+### Extensions and project integrations
+
+When using IREE within other projects, you can register compiler plugins and
+runtime HAL drivers. You can also bring your own copy of LLVM and some other
+tools. See the root
+[`CMakeLists.txt`](https://github.com/openxla/iree/blob/main/CMakeLists.txt)
+for details.
```

Comment:
Also not sure about this new section. Could be useful to a few people to have this advertised up front though.

Reply:
I like this testing section and the "Optional components" section above. It's useful info, and the section titles and ordering make it clear that it's not necessary for building.